migrations/Version202109040100000000_taoLti.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace oat\taoLti\migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\Exception\IrreversibleMigration;
+use oat\tao\scripts\SyncModels;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+
+final class Version202109040100000000_taoLti extends AbstractMigration
```

Comment:
Seems to be a custom version number, if you'll need generate another migration you can run f.e.: `docker exec -it TAO_PHPFPM_CONTAINER php index.php 'oat\tao\scripts\tools\Migrations' -c generate -e taoLti`